General.java

```diff
@@ -0,0 +1,55 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class General {
+    //Create a function that accepts an array of integers and returns the sum of all the even numbers in the array.
+    public static Integer question1(List<Integer> numbers){
+        int sum = 0;
+        for (Integer i : numbers){
```

Comment:
Good implementation but does not check for even numbers as the question asks use modulo for this